pkgs/applications/networking/protonmail-bridge/default.nix

```diff
@@ -1,73 +1,43 @@
-{ stdenv, fetchurl, lib, qtbase, qtmultimedia, qtsvg, qtdeclarative, qttools, qtgraphicaleffects, qtquickcontrols2, full
-, libsecret, libGL, libpulseaudio, glib, wrapQtAppsHook, mkDerivation }:
+{ lib, buildGoModule, fetchFromGitHub, pkgconfig, libsecret }:
 
-let
-  version = "1.2.3-1";
-
-  description = ''
-    An application that runs on your computer in the background and seamlessly encrypts
-    and decrypts your mail as it enters and leaves your computer.
-
-    To work, gnome-keyring service must be enabled.
-  '';
-in mkDerivation {
+buildGoModule rec {
   pname = "protonmail-bridge";
-  inherit version;
+  version = "1.4.5";
 
-  src = fetchurl {
-    url = "https://protonmail.com/download/protonmail-bridge_${version}_amd64.deb";
-    sha256 = "032ggk9fvd19fbsqkzwzwh0hpyg8gpkrin71di7zsx6ias5innw1";
+  src = fetchFromGitHub {
+    owner = "ProtonMail";
+    repo = "proton-bridge";
+    rev = "br-${version}";
+    sha256 = "sha256-J7m1JcLrjUw0PIMoRWDXH1hax97/C7zq6Rj2o/SAaYw=";
   };
 
-  sourceRoot = ".";
+  vendorSha256 = "sha256-goaMQ4wmUoc/TLsIxmwH9C2VtKsnuwRmAs9C1sGosk0=";
 
-  unpackCmd = ''
-    ar p "$src" data.tar.xz | tar xJ
-  '';
+  nativeBuildInputs = [ pkgconfig ];
 
-  installPhase = ''
-    mkdir -p $out/{bin,lib,share}
+  buildInputs = [ libsecret ];
 
-    cp -r usr/lib/protonmail/bridge/protonmail-bridge $out/lib
-    cp -r usr/share $out
-
-    ln -s $out/lib/protonmail-bridge $out/bin/protonmail-bridge
+  buildPhase = ''
+    make build-nogui
   '';
 
-  postFixup = let
-    rpath = lib.makeLibraryPath [
-      stdenv.cc.cc.lib
-      qtbase
-      qtquickcontrols2
-      qtgraphicaleffects
-      qtmultimedia
-      qtsvg
-      qtdeclarative
-      qttools
-      libGL
-      libsecret
-      libpulseaudio
-      glib
-    ];
-  in ''
-    patchelf \
-      --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" \
-      --set-rpath "${rpath}" \
-      $out/lib/protonmail-bridge
-
-    substituteInPlace $out/share/applications/ProtonMail_Bridge.desktop \
-      --replace "/usr/" "$out/" \
-      --replace "Exec=protonmail-bridge" "Exec=$out/bin/protonmail-bridge"
+  installPhase = ''
+    install -Dm555 Desktop-Bridge $out/bin/protonmail-bridge
   '';
 
-  buildInputs = [ qtbase qtquickcontrols2 qtmultimedia qtgraphicaleffects qtdeclarative ];
-
-  meta = with stdenv.lib; {
-    homepage = "https://www.protonmail.com/bridge";
-    license = licenses.mit;
+  meta = with lib; {
+    homepage = "https://github.com/ProtonMail/proton-bridge";
+    changelog = "https://github.com/ProtonMail/proton-bridge/blob/master/Changelog.md";
+    downloadPage = "https://github.com/ProtonMail/proton-bridge/releases";
+    license = licenses.gpl3;
     platforms = [ "x86_64-linux" ];
     maintainers = with maintainers; [ lightdiscord ];
+    description = "Bridge for e-mail clients";
```

Comment:
Doesn't it only work with protonmail.com?

Reply:
Exactly, should I write something like "ProtonMail Bridge for e-mail clients" ? But in this case it may conflict with the requirement of [meta.description should not start with the package name](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes) ?

Reply:
How about?
```
Integrates your ProtonMail account with your local email client
```